lib/Service/ProjectService.php

```diff
@@ -3613,6 +3616,112 @@ public function cronRepeatBills(?int $billId = null): array {
 		return $result;
 	}
 
+	private function copyBillPaymentMethodOver (string $projectid, array $bill, string $toprojectid): int {
```

Comment:
```suggestion
	private function copyBillPaymentMethodOver (string $projectid, array $bill, string $toProjectId): int {
```
I know it's not used everywhere yet but let's use camel case.

Reply:
👍 I've fixed it on all the places I wrote new code for the project